docs/bootstrapper.md

```diff
@@ -0,0 +1,64 @@
+# QGB bootstrapper
+
+To bootstrap the QGB P2P network, we use the bootstrapper QGB node type to accept connections from freshly created orchestrators/relayers and share its peer table with them.
+
+## How to run
+
+### Install the QGB binary
+
+Make sure to have the QGB binary installed. Check [here](https://docs.celestia.org/nodes/qgb-binary) for more details.
+
+### Init the store
+
+Before starting the bootstrapper, we will need to init the store:
+
+```ssh
+qgb bootstrapper init
+```
+
+By default, the store will be created un `~/.bootstrapper`. However, if you want to specify a custom location, you can use the `--home` flag. Or, you can use the following environment variable:
+
+| Variable            | Explanation                         | Default value     | Required |
+|---------------------|-------------------------------------|-------------------|----------|
+| `BOOTSTRAPPER_HOME` | Home directory for the bootstrapper | `~/.bootstrapper` | Optional |
+
+### Add keys
+
+In order for the bootstrapper to start, it will need a P2P private key in store.
+
+The P2P private key is optional, and a new one will be generated automatically on the start if none is provided.
```

Comment:
these sentences are a bit confusing

it needs...
... is optional

perhaps we can just drop the first sentence

Reply:
cc @SweeXordious since it auto merged